### DISCUSSION TITLE
Welcome to gamedao-protocol Discussions!

### DISCUSSION THREAD

**Discussion by 2075** [gamedaoco/gamedao-protocol#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 4eckd** [comment]
👋 hey,

My name is Jesse but please call me J. 

In a web 3 Fullstack engineer and have been for around 10 years and another 5 years working in validation and IT systems.

I'm particularly interested in gaming space tech and always have been. I think many young tech pros probably begin their journeys in this immersive tech rich space. I've carried on into cryptography and probably fair mechanisms now for a few years because of the igaming industry where I have worked in as well for approximately as much. 

I want to dedicate more time diving into any areas of potential interest where I might fit as well as find grant proposal or professional work contributions. 

I live in Oakland California if that matters. And no, I cannot work full time in office environment. You'll be lucky to have me more than one day per week and it's only because this work demands focus if you are a project manager or senior engineer. However, I'm open to something if the expectations are also in alignment and clear. 

Happy to chat more,
J.
